lnbits/nodes/cln.py

```diff
@@ -310,10 +310,10 @@ async def get_invoices(
         return Page(
             data=[
                 NodeInvoice(
-                    bolt11=invoice["bolt11"],
-                    amount=invoice["amount_msat"],
+                    bolt11=invoice.get("bolt11") if invoice.get("bolt11") != None else invoice.get("bolt12"),
```

Comment:
Unfortunately NodeInvoice has a bolt11 field, but no bolt12 field. Now I shoe-horn a bolt12 onto a field named "bolt11".

I suggest renaming NodeInvoice.bolt11 to NodeInvoice.bolt1x or something. Another approach is to use two fields in NodeInvoice, but that unnecessarily complicates things, I think.

Reply:
I think a seperate field would be the best way. We'd also have to consider how to show that in the ui aswell

Reply:
There are so many places where the code expects `bolt11` to be set, I found it extremely difficult to refactor it now. I think that as a short-term/mid-term fix this will have to do.